api/s2n.h

```diff
@@ -28,7 +28,7 @@
      */
     #define S2N_API __attribute__((visibility("default")))
 #else
-    /**
+/**
      * Marks a function as belonging to the public s2n API.
```

Comment:
What did you do to this file's formatting O_o

Reply:
Ah!